cmd/mount_all.go

```diff
@@ -264,7 +265,8 @@ func filterAllowedContainerList(containers []string) []string {
 // mountAllContainers : Iterate allowed container list and create config file and mount path for them
 func mountAllContainers(containerList []string, configFile string, mountPath string, configFileExists bool) error {
 	// Now iterate filtered container list and prepare mount path, temp path, and config file for them
-	fileCachePath := viper.GetString("file_cache.path")
+	fileCachePath := ""
```

Comment:
is there any test we can add for this fix?

Reply:
We can not because if you provide the path it will be set, if you dont then it will be empty. In no case its erroring out.
In case viper.GetString it used to fail if path is given on cli and work if it's given in config file.